docs/bugs.json

```diff
@@ -1,4 +1,15 @@
 [
+    {
+        "uid": "SOL-2022-2",
+        "name": "AbiEncodeFromCalldataNestedArraySizeValidationBug",
```

Comment:
My naming suggestion from the chat:
```suggestion
        "name": "AbiEncodeNestedCalldataArraySizeValidation",
```

Reply:
I went for the later suggestion of ``NestedCallataArrayAbiReencodingSizeValidation`` for now. Open to changing it further.